src/App.js

```diff
@@ -3,12 +3,14 @@ import { useState, useEffect } from 'react';
 import { Route, Routes } from 'react-router-dom';
 import Header from './Header';
 import Gallery from './Gallery';
+import Footer from './Footer';
 
 function App() {
   return (
     <div className="App">
       <Header />
```

Comment:
perhaps consider using some semantic elements like <header>, <main> & <section> tags over divs.  Would be better for accessibility scores (screen readers).